CMakeLists.txt

```diff
@@ -45,26 +45,24 @@ function(openpmd_option name description default)
     set(openPMD_CONFIG_OPTIONS ${openPMD_CONFIG_OPTIONS} ${name} PARENT_SCOPE)
 endfunction()
 
-openpmd_option(MPI              "Enable MPI support"                        AUTO)
-openpmd_option(HDF5             "Enable HDF5 support"                       AUTO)
-openpmd_option(ADIOS1           "Enable ADIOS1 support"                     AUTO)
-openpmd_option(ADIOS2           "Enable ADIOS2 support"                     OFF)
-#openpmd_option(JSON             "Enable JSON support"                      AUTO)
-openpmd_option(INVASIVE_TESTS   "Enable unit tests that modify source code" AUTO)
-openpmd_option(PYTHON           "Enable Python bindings"                    AUTO)
+openpmd_option(MPI            "Enable MPI support"                        AUTO)
+openpmd_option(HDF5           "Enable HDF5 support"                       AUTO)
+openpmd_option(ADIOS1         "Enable ADIOS1 support"                     AUTO)
+openpmd_option(ADIOS2         "Enable ADIOS2 support"                     OFF)
+#openpmd_option(JSON           "Enable JSON support"                      AUTO)
+openpmd_option(INVASIVE_TESTS "Enable unit tests that modify source code" AUTO)
+openpmd_option(PYTHON         "Enable Python bindings"                    AUTO)
 
 option(openPMD_USE_INTERNAL_VARIANT "Use internally shipped MPark.Variant" ON)
-option(openPMD_USE_INTERNAL_CATCH   "Use internally shipped Catch2" ON)
+option(openPMD_USE_INTERNAL_CATCH   "Use internally shipped Catch2"        ON)
+
+option(openPMD_ENABLE_VERIFY "Enable internal VERFIY macro" ON)
```

Comment:
- typo: "verify" macro
- can you use `openPMD_USE_VERIFY` so we have it consistently named?
- can we write "Keep VERIFY (assert) macros enabled in release mode"?